py/mpconfig.h

```diff
@@ -1372,6 +1372,13 @@ typedef double mp_float_t;
 #define MICROPY_PY_SYS_EXC_INFO (0)
 #endif
 
+// Whether to provide "sys.executable", which is the absolute path to the
+// micropython binary
```

Comment:
Missing dot after binary?

Reply:
None of the comments in this file end in full stop.

(I had to restrain myself!!)

Reply:
Oh, ok then :)